build.sbt

```diff
@@ -110,7 +110,6 @@ lazy val core = project
     metaMacroSettings,
     moduleName := "scalafmt-core",
     test in assembly := {
-      (test in Test).value
```

Comment:
The task runs, so the SBT warning is wrong, but I ever run root/assembly anymore so I'm OK with dropping this line.